### ISSUE TITLE
No async/await highliting for jsx code

### ISSUE THREAD

**Issue by 0e39bf7b** [gohugoio/hugo#7520]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
0.74.3
</pre>

### Does this issue reproduce with the latest release?
yes

After upgrade from 0.55.1 to 0.74.3 syntax highlight becomes different. I have such jsx code:

```jsx
class UserProvider extends React.Component {
  state = { user: null };

  async componentDidMount() {
    const user = await fetchUser();
    this.setState({ user });
  }

  render() {
    return this.props.render(this.state.user);
  }
}
```

In 0.55.1 it looks like this:

![Screenshot 2020-07-28 at 10 51 22](https://user-images.githubusercontent.com/49288816/88635483-5ebd8b80-d0c0-11ea-8b0a-fd55713680e3.png)

In 0.56 and later versions it looks like this:

![Screenshot 2020-07-28 at 10 51 32](https://user-images.githubusercontent.com/49288816/88635567-79900000-d0c0-11ea-99b3-e8781e6fc3ba.png)

As you can see there is no highlighting for async/await keywords. Is it a bug or I'm doing something wrong?

**Comment by jmooring** [comment]
Created issue and submitted PR upstream:
<https://github.com/alecthomas/chroma/issues/385>

**Comment by jmooring** [comment]
Fixed with 4318dc7. Please close.
